client/my-sites/site-settings/settings-security/test/main.jsx

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
These throw a warning because `requestSupportHistory` is missing an `onError` handler:
https://github.com/Automattic/wp-calypso/blob/e314a251e948361d9019441d2976041f51fed054/client/state/data-layer/wpcom/help/support-history/index.js#L27-L30
But it seems like we have a lot of tests that do, and I'd prefer not to change unrelated code in this PR.

Reply:
Thanks, opened an issue here: https://github.com/Automattic/wp-calypso/issues/54663